plugins/fabrik_element/cascadingdropdown/cascadingdropdown.php

```diff
@@ -824,10 +824,26 @@ protected function buildQuery($data = array(), $incWhere = true, $opts = array()
 			{
 				foreach ($whereval as &$v)
 				{
-					$v = $db->quote($v);
-				}
 
-				$where .= count($whereval) == 0 ? '1 = -1' : $wherekey . ' IN (' . implode(',', $whereval) . ')';
+					// Jaanus: Solving bug: imploded arrays when chbx in repeated group         
+					
+					if (is_array($v)) 
```

Comment:
no need to test for an array - safeQuote() will parse $v if its an array or a string value